network-api/networkapi/wagtailpages/pagemodels/publications/publication.py

```diff
@@ -110,12 +111,21 @@ def is_chapter_page(self):
         parent = self.get_parent().specific
         return parent.__class__ is PublicationPage
 
+    @property
+    def zen_nav(self):
+        return True
+
     def breadcrumb_list(self):
         """
         Get all the parent PublicationPages and return a QuerySet
         """
         return Page.objects.ancestor_of(self).type(PublicationPage).live()
 
+    def get_context(self, request, *args, **kwargs):
+        context = super().get_context(request, *args, **kwargs)
+        # menu_items is required for zen_nav in the templates
+        context['menu_items'] = Homepage.objects.first().get_children().live().in_menu()
```

Comment:
Adds "menu_items" into the page context so the zen_nav has menu items to show.

Reply:
Is this something we can utilize here? https://github.com/mozilla/foundation.mozilla.org/blob/master/network-api/networkapi/wagtailpages/utils.py#L32-L46

e.g., 
```py
def get_context(self, request):
        context = super().get_context(request)
        return set_main_site_nav_information(self, context, 'Homepage')
```